unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs

```diff
@@ -59,6 +61,37 @@ public static void CleanMaterials(Renderer r)
             }
         }
 
+        public static ScriptableRendererFeature ToggleRenderFeature<T>(this UniversalRenderPipelineAsset asset, bool enable) where T : ScriptableRendererFeature
+        {
+            var type = asset.GetType();
+            var propertyInfo = type.GetField("m_RendererDataList", BindingFlags.Instance | BindingFlags.NonPublic);
```

Comment:
This is quite expensive, I know there's no other way around it but let's use this wisely!

Reply:
Yeah I don't think we will have many use cases of this, and probably if it will be used it won't be used often at runtime